website/content/docs/enterprise/pkcs11-provider/aws-xks.mdx

```diff
@@ -0,0 +1,201 @@
+---
+layout: docs
+page_title: AWS KMS External Key Store (XKS) - PKCS#11 Provider - Vault Enterprise
+description: |-
+  The Vault PKCS#11 Provider can be used with AWS KMS External Key Store.
+---
+
+# Vault with AWS KMS External Key Store (XKS) via PKCS#11 and XKS Proxy
+
+~> **Important**: AWS [`xks-proxy`](https://github.com/aws-samples/aws-kms-xks-proxy) is not considered production-ready.
```

Comment:
I see it's the reference implementation, but I don't see any mention on that page or the blog of not being production-ready. Am I missing it, or is there another page which references the production readiness directly?

Reply:
nit: For consistency with other callouts? I guess the formatting already implies it's important.

```suggestion
~> **Note**: AWS [`xks-proxy`](https://github.com/aws-samples/aws-kms-xks-proxy) is not considered production-ready.
```

Reply:
I don't know if they have called out the production readiness of it explicitly or not.

Reply:
Should we delete the callout then?

Reply:
:+1: